codalab/lib/upload_manager.py

```diff
@@ -4,6 +4,9 @@
 
 from codalab.common import UsageError
 from codalab.lib import crypt_util, file_util, path_util, zip_util
+from codalab.lib.beam.filesystems import FileSystems
+
+# TODO: fix methods here.
```

Comment:
What's this?  explain more?

Reply:
Some of my comments / debugging statements are outdated 🙂 -- I'll update / remove them.